ui/vue.config.js

```diff
@@ -16,5 +18,18 @@ module.exports = {
     },
     "transpileDependencies": [
         "vuetify"
-    ]
+    ],
+    configureWebpack: {
+        plugins: [
+            new CopyPlugin({
+                patterns: [
+                    {
+                        from: 'node_modules/swagger-ui/dist/oauth2-redirect.html',
+                        flatten: true,
+                        force: true
```

Comment:
At build time, the `oauth2-redirect.html` static file that is included in the swagger-ui `dist` is copied into our own `dist` so it gets hosted like everything else.

I argue this is the best solution because:
- It is very much recommended to `import` other things from their `dist`, like the .css files to style the UI
- We can't just `import` this HTML like Gatsby users etc can because we don't want to mess with our router to treat this like a Vue template (in other words, we'd have to mess with Webpack and our router to be able to `import` HTML without changing it)
- CopyPlugin will error the build if this file ever disappears
- Having an automated build-time copy avoids licensing issues and means they can update that file whenever they like

Reply:
makes sense to me